packages/ERTP/src/types.js

```diff
@@ -53,7 +53,7 @@
  *
  * @callback AmountMakeBrandValue
  * @param {Brand} brand
- * @param {Value} allegedValue
+ * @param {Value | number} allegedValue
```

Comment:
Please don't change this. Numbers are deprecated and not allowed in the linting.